src/alphafold_impact/pipelines/data_processing_nih_clinical_trials/__init__.py

```diff
@@ -0,0 +1,10 @@
+"""
```

Comment:
We should do something about the `__init__.py` docstrings :)

Reply:
How about removing them all?